src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -1065,69 +1102,136 @@ private void biteBlock(@Nonnull BlockPos pos, @Nonnull IBlockState blockState, @
             NBTTagCompound updatePacket = bloodStats.writeUpdate(new NBTTagCompound());
             sync(updatePacket, true);
 
-            biteCooldown = Balance.vp.BITE_COOLDOWN;
 
         }
 
     }
 
+    /**
+     * Bite the entity with the given id.
+     * Checks reach distance
+     *
+     * @param entityId The id of the entity to start biting
+     */
+    public void biteEntity(int entityId) {
+        Entity e = player.getEntityWorld().getEntityByID(entityId);
+        if (player.isSpectator()) {
+            VampirismMod.log.w(TAG, "Player can't bite in spectator mode");
+            return;
+        }
+        if (e != null && e instanceof EntityLivingBase) {
+            if (e.getDistance(player) <= player.getEntityAttribute(EntityPlayer.REACH_DISTANCE).getAttributeValue() + 1) {
+                if (determineBiteType(((EntityLivingBase) e)) == BITE_TYPE.ATTACK || victim_bite_type == BITE_TYPE.ATTACK_HUNTER) {
```

Comment:
I think the second check (==ATTACK_HUNTER) does not respect the new value, but a older value previously stored in victim_bite_type.

Also bite type is determined up to two times in the whole method. Probably better to get it only once in the beginning (Store it either directly in victim_bite_type or in some intermediate variable)